src/main/java/com/amazon/opendistroforelasticsearch/ad/model/ADTaskType.java

```diff
@@ -26,7 +26,25 @@
 
 package com.amazon.opendistroforelasticsearch.ad.model;
 
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
 public enum ADTaskType {
+    // TODO: remove these old task types
     REALTIME,
-    HISTORICAL
+    HISTORICAL,
+    REALTIME_SINGLE_ENTITY,
+    REALTIME_HC_DETECTOR,
+    HISTORICAL_SINGLE_ENTITY,
+    HISTORICAL_HC_DETECTOR,
+    HISTORICAL_HC_ENTITY;
```

Comment:
There is no "REALTIME_HC_ENTITY", right?

Reply:
Will remove `REALTIME` and `HISTORICAL`.